packages/11ty/content/_assets/javascript/application/index.js

```diff
@@ -190,6 +190,7 @@ function scrollToHash() {
   // Select all links with hashes
   $('a[href*="#"]')
     // Remove links that don't actually link to anything
+    .not('.q-figure__modal-link')
```

Comment:
sort of a hack to prevent figure modal links from adding hashes to `window.location` and triggering scroll-to-hash behavior